lib/gunit/msvc15/gtest/gtest.vcproj

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="Windows-1252"?>
```

Comment:
Are you sure that we actually need those solution/project files? Why do we need them? Is it just because of version of MSVC they were created with?
Michał Cłapiński claims that he has successfully compiled old projects with MSVC 2015 without converting them (he forced platform toolset version with the following command line option: `/p:PlatformToolset=v140`. Have you investigated that?